first_order_model/reconstruction.py

```diff
@@ -233,20 +246,17 @@ def reconstruction(config, generator, kp_detector, checkpoint, log_dir, dataset,
                 driving = frame_to_tensor(img_as_float32(frame), device)
                 
                 # get LR video frame
-                driving_lr = F.interpolate(driving, 64).data.cpu().numpy()
-                driving_lr = np.transpose(driving_lr, [0, 2, 3, 1])[0]
-                driving_lr *= 255
-                driving_lr = driving_lr.astype(np.uint8)
+                driving_lr = resize_tensor_to_array(driving, lr_size, device)
                 
                 driving_lr_av = av.VideoFrame.from_ndarray(driving_lr)
                 driving_lr_av.pts = av_frame.pts
                 driving_lr_av.time_base = av_frame.time_base
                 
-                driving_lr, compressed_tgt = get_frame_from_video_codec(driving_lr_av, lr_encoder, lr_decoder, 5)
+                driving_lr, compressed_tgt = get_frame_from_video_codec(driving_lr_av, lr_encoder, lr_decoder, 48)
```

Comment:
This should be irrelevant in the current master, please be very careful when fixing merge conflicts. I'd first pull in current master and send a new PR that has fewer of these now stale changes.

Reply:
why do you say it's irrelevant?

Reply:
I'd just use the one that is in master, not tweak them

Reply:
(It is outdated as per this PR also)

Reply:
fixed, using the one in master